src/server/utils/cdnCache.js

```diff
@@ -26,15 +25,9 @@ export const oneAppModuleCachePath = path.join(oneAppDirectoryPath, cacheFileNam
 export const showCacheInfo = async () => {
   try {
     const stats = await fsPromises.stat(oneAppModuleCachePath);
-    const fileSizeOnMB = stats.size / (1024 * 1024); // bytes to mb
-    const message = `File size of ${cacheFileName}: ${chalk.bold.greenBright(fileSizeOnMB.toFixed(2), 'MB')}`;
-    const separator = '*'.repeat(message.length);
-    console.log(chalk.bold.cyanBright(separator));
-    console.log(chalk.bold.cyanBright('CACHE INFORMATION'));
-    console.log(message);
-    console.log('To clear the cache, please delete this file:');
-    console.log(`    ${chalk.bold.cyanBright(path.join('~', oneAppDirectoryName, cacheFileName))}`);
-    console.log(chalk.bold.cyanBright(separator));
+    const fileSizeInMB = stats.size / (1024 * 1024); // bytes to mb
+    const cachePath = path.join('~', oneAppDirectoryName, cacheFileName);
+    console.info('Local module cache size is %sMB. To clear the cache, delete %s', fileSizeInMB.toFixed(2), cachePath);
```

Comment:
should this be info? I feel like as this is a development only addition it makes sense to keep this logged out during the beginning especially as it is only logged once.

Reply:
this one could be a good candidate for the new logger.dev logging method